src/Cech_complex/include/gudhi/Cech_complex/Cech_kernel.h

```diff
@@ -0,0 +1,107 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hind Montassif
+ *
+ *    Copyright (C) 2021 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef CECH_KERNEL_H_
+#define CECH_KERNEL_H_
+
+#include <CGAL/Epeck_d.h>
+
+#include <cmath>  // for std::sqrt
+
+namespace Gudhi {
+
+// namespace cech_complex {
+
+/** @brief Compute the radius of the minimal enclosing ball between Points given by a range of coordinates.
+ * The points are assumed to have the same dimension. */
+class Minimal_enclosing_ball_radius {
+ public:
+   /** \brief Enclosing ball radius from two points using CGAL.
+   *
+   * @param[in] point_1
+   * @param[in] point_2
+   * @return Enclosing ball radius for the two points.
+   * \tparam Point must be a Kernel::Point_d from CGAL.
+   *
+   */
+  template< typename Kernel = CGAL::Epeck_d<CGAL::Dynamic_dimension_tag>,
```

Comment:
This pretty much hardcodes the kernel, why not make it a parameter of the class?